src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ CHANGELOG
    the transport. See `ConsumeMessagesCommand`.
  * The optional `$busNames` constructor argument of the class `ConsumeMessagesCommand` was removed.
  * [BC BREAK] 2 new methods were added to `ReceiverInterface`:
-   `ack()` and `reject()`.
+   `ack()` and `reject()`. Also `receive()` was changed to `get()`
+   and `stop()` was removed.
```

Comment:
`receive` was not really changed to `get`, was it? It's more like `received` was removed and `get` added, right?